### PR TITLE
Get texture by `source` or `video` props

### LINKED ISSUE #121: getTextureFromProps / args
### Description
Discussion is here: https://github.com/inlet/react-pixi/pull/116#discussion_r301320513

### Steps to reproduce
n/a

### Additional info

- @inlet/react-pixi version: 0.7.1
- React version: 16.8.6
- ReactDOM version: 16.8.6
- PIXI version: 5.0.4

### PR DESCRIPTION
**Pull request by inlet**
Get texture from `source` and `video` props.

Fixes #121

### CODE REVIEW

File: src/utils/props.js [hunk 1]
```diff
@@ -51,28 +51,35 @@ export const PROPS_DISPLAY_OBJECT = {
  * Can be either texture or image
  *
  * @param {string} elementType
- * @param {PIXI.Texture|undefined} texture
- * @param {string|undefined} image
- * @param {boolean|undefined} crossorigin
- * @returns {PIXI.Texture}
+ * @param {object} props
```
Comment (fabienjuif) [inline src/utils/props.js]: maybe you can specify props like I did in my other PR?
#125

Comment (inlet) [reply src/utils/props.js]: I intentionally removed them as these are all defined in the new TS definition which gives way better type support

Comment (inlet) [reply src/utils/props.js]: Even if you're not using typescript (type definitions are automatically picked up in VSCode and many other editors)

Comment (fabienjuif) [reply src/utils/props.js]: You are right 👍

**Pull request merged**